unison-src/transcripts/add-run.md

```diff
@@ -0,0 +1,59 @@
+# add.run
+
+```ucm:hide
+.> builtins.merge
+```
+
+## happy path
+```unison
+even : Nat -> Boolean
+even x = if x == 0 then true else odd (drop x 1)
+
+odd : Nat -> Boolean
+odd x = if x == 0 then false else even (drop x 1)
+
+is2even : 'Boolean
+is2even = '(even 2)
+```
+
+```ucm:error
+.> add.run foo
+```
+
+```ucm
+.> run is2even
+```
+
+```ucm:error
+.> add.run is2even
+```
+
+```ucm
+.> add.run foo.bar.baz
+```
+
+```ucm
+.> view foo.bar.baz
+```
+
+## It continues to work if a dependency is changed
+
+```unison
+unique type Foo = Foo | Bar
+
+foo : 'Foo
+foo = 'Foo
+```
+
+```ucm
+.> run foo
+```
+
+```unison
+unique type Foo = Foo | Bar | Baz
+```
+
+```ucm
+.> add.run result-foo
+.> view Foo
+```
```

Comment:
Due to the `slurp` error I comment on earlier in the PR, if you repeat this step one more time you get pretty unexpected results, e.g. add this to the end of this transcript:

    ```unison
    unique type Foo = Foo | Bar | Baz | Bingo

    foo : 'Foo
    foo = 'Foo
    ```

    ```ucm
    .> run foo
    .> add.run result-foo2
    .> view Foo
    .> view result-foo2
    ```


And you get:

    ```unison
    unique type Foo = Foo | Bar | Baz | Bingo

    foo : 'Foo
    foo = 'Foo
    ```

    ```ucm

      I found and typechecked these definitions in scratch.u. If you
      do an `add` or `update`, here's how your codebase would
      change:
      
        ⍟ These new definitions are ok to `add`:
        
          foo : 'Foo
        
        ⍟ These names already exist. You can `update` them to your
          new definition:
        
          unique type Foo

    ```
    ```ucm
    .> run foo

      Foo

    .> add.run result-foo2

    .> view Foo

      unique type Foo = Foo | Bar

    .> view result-foo2

      ⚠️
      
      The following names were not found in the codebase. Check your spelling.
        result-foo2

    ```



    🛑

    The transcript failed due to an error in the stanza above. The error is:


      ⚠️
      
      The following names were not found in the codebase. Check your spelling.
        result-foo2